packages/datetime/test/dateInputTests.tsx

```diff
@@ -11,24 +11,31 @@ import * as sinon from "sinon";
 
 import { Classes as CoreClasses, InputGroup, Keys, Popover, Position } from "@blueprintjs/core";
 import { Months } from "../src/common/months";
+import { IDateFormatter } from "../src/dateFormatter";
 import { Classes, DateInput, TimePicker, TimePickerPrecision } from "../src/index";
 import * as DateTestUtils from "./common/dateTestUtils";
 
+const format: IDateFormatter = {
+    dateToString: date => date.toLocaleString(),
+    placeholder: "enter date",
+    stringToDate: str => new Date(Date.parse(str)),
+};
```

Comment:
big problems with this formatter. `Date.parse(str)` uses UTC timezone for "YYYY-MM-DD" strings, so some dates come out a day earlier than expected. should I just use a moment formatter in these tests?

Reply:
```js
new Date("2/1/2015")
> "Sun Feb 01 2015 00:00:00 GMT-0800 (PST)"

new Date("2015-02-01")
> "Sat Jan 31 2015 16:00:00 GMT-0800 (PST)"
```